test-suite-kotlin/src/test/kotlin/io/micronaut/docs/context/events/SampleEventEmitterBean.kt

```diff
@@ -24,7 +24,7 @@ import jakarta.inject.Singleton
 class SampleEventEmitterBean {
```

Comment:
The groovy and java ones also need updating

Reply:
done